src/Components/ArticlePage.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router";
+import axios from "axios";
+
+const ArticlePage = () => {
+  const { article_id } = useParams(); 
+  const [article, setArticle] = useState(null);
+  const [loading, setLoading] = useState(true);
```

Comment:
this should start as false and we set it to true as the first thing in our useEffect